server/service/transport_error.go

```diff
@@ -182,3 +187,21 @@ func encodeError(ctx context.Context, err error, w http.ResponseWriter) {
 		enc.Encode(je)
 	}
 }
+
+func sendToSentry(ctx context.Context, err error) {
+	v, haveUser := viewer.FromContext(ctx)
+	h, haveHost := host.FromContext(ctx)
+	localHub := sentry.CurrentHub().Clone()
```

Comment:
Is this okay if it's not been configured, or do we need some check here?

Reply:
Yes, local tests seem to work (fail) as expected without sentry.

Reply:
IMO it would be less error prone if we don't run any of this code when sentry is not enabled.

Reply:
Made it so it only calls `sendToSentry` when it's enabled. As for the calls to `sentry.CaptureException` they return early when the internal sentry client is `nil`, which is the case when sentry is not configured and `sentry.Init` is not called.

Adding logic to only call `sentry.CaptureException` when sentry is configured would be basically duplicating this logic in our layer, which feels redundant.